cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -0,0 +1,172 @@
+// React imports
+import React, { ChangeEvent, useState } from 'react';
+
+// Library imports
+import { ClarityIcons, blockIcon, blocksGroupIcon, clusterIcon } from '@cds/core/icon';
+import { CdsControlMessage, CdsFormGroup } from '@cds/react/forms';
+import { CdsInput } from '@cds/react/input';
+import { useForm } from 'react-hook-form';
+import { CdsRadio, CdsRadioGroup } from '@cds/react/radio';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsButton } from '@cds/react/button';
+
+// App imports
+import { INPUT_CHANGE } from '../../../../state-management/actions/Form.actions';
+import { StepProps } from '../../../../shared/components/wizard/Wizard';
+
+ClarityIcons.addIcons(blockIcon, blocksGroupIcon, clusterIcon);
+
+interface FormInputs {
+    CLUSTER_NAME: string;
+    CLUSTER_SERVICE_CIDR: string;
+    CLUSTER_POD_CIDR: string;
+}
+
+const unmanagedClusterProviders = [
+    {
+        label: 'calico',
+        value: 'CALICO',
+    },
+    {
+        label: 'anthrea',
+        value: 'ANTHREA',
```

Comment:
Please check this minor typo that needs correction
```suggestion
        label: 'antrea',
        value: 'ANTREA',
```

Reply:
Oh Sorry I think it was written that way in the Figma files. Thank you!